### PR TITLE
577 Sighting POST support removed

### PR DESCRIPTION
**Pull request by hwindsor**
## Pull Request Overview

- Sightinng POST support removed
- Test updated to make sure
---

### CODE REVIEW

File: app/modules/asset_groups/models.py [hunk 1]
```diff
@@ -186,10 +186,14 @@ def commit(self):
...
         try:
+            # Don't send the encounter guids. EDM doesn't use them but they confuse anyone reading logs
+            sent_data = self.config
+            for enc in sent_data['encounters']:
+                enc.pop('guid')
             result_data = current_app.edm.request_passthrough_result(
                 'sighting.data',
                 'post',
-                {'data': self.config, 'headers': {'Content-Type': 'application/json'}},
+                {'data': sent_data, 'headers': {'Content-Type': 'application/json'}},
```
Comment (colinwkingen) [inline app/modules/asset_groups/models.py:196]: Huh. Did we need them before for some reason or were they always extraneous?

**Review by colinwkingen — APPROVED** [review]
LGTM

File: app/modules/asset_groups/models.py (hunk 1 above)
Comment (hwindsor) [reply app/modules/asset_groups/models.py:196]: They were always extraneous. These were the temporary encouner IDs that could be used to update the encounter metadata in the AGS before the real one was created.

**Pull request merged**